code/game/turfs/open/floor/plating/misc_plating.dm

```diff
@@ -271,13 +271,37 @@
 /turf/open/floor/plating/grass
 	name = "grass"
 	desc = "A patch of grass."
-	icon_state = "grass"
-	broken_states = list("sand")
+	icon_state = "grass0"
+	var/smooth_icon = 'icons/turf/floors/grass.dmi'
```

Comment:
But new variables under the redefined ones. In this case, put this variable under the `layer` one.